### PR TITLE
feat: See E2EI certificate details [WPB-2213]

### PR DESCRIPTION
**Pull request by przemvs**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-2213" title="WPB-2213" target="_blank">EpicWPB-2213</a>  5.3.25 See E2EI certificate details
  </td></table>
  <br />
 

## Description
Showing modal with PEM certificate chain. You can Download ( it will be prepared in another task ) or Copy to Clipboard this certificate.

* For now I'm using **TMP_CERTIFICATE** variable for display certificate, it will be removed while we have working functionality.

## Screenshots/Screencast (for UI changes)
image

## Checklist

- [ ] PR has been self reviewed by the author;
- [x] Hard-to-understand areas of the code have been commented;
- [x] If it is a core feature, unit tests have been added;|

### CODE REVIEW

File: src/script/components/Modals/CertificateDetailsModal/CertificateDetailsModal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
```
Comment (V-Gira) [inline src/script/components/Modals/CertificateDetailsModal/CertificateDetailsModal.test.tsx]:
```suggestion
 * Copyright (C) 2023 Wire Swiss GmbH
```
got em

File: src/style/modal/certificate-details-modal.less [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
```
Comment (V-Gira) [inline src/style/modal/certificate-details-modal.less]:
```suggestion
 * Copyright (C) 2023 Wire Swiss GmbH
```

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
     "@datadog/browser-rum": "^4.47.0",
     "@emotion/react": "11.11.1",
     "@loadable/component": "^5.15.3",
+    "@wireapp/api-client": "file:.yalc/@wireapp/api-client",
```
Comment (atomrc) [inline package.json]: This is wrong, should not be commited

File: src/style/foundation/main.less [hunk 4]
```diff
@@ -126,3 +126,4 @@
 @import '../modal/service-modal';
 @import '../modal/legal-hold-modal';
 @import '../modal/invite-modal';
+@import '../modal/certificate-details-modal';
```
Comment (atomrc) [inline src/style/foundation/main.less]: Can we move that to a `styles.ts` file right next to the component?

**Review by atomrc — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (przemvs) [reply package.json]: [795a769](https://github.com/wireapp/wire-webapp/pull/15745/commits/795a76903f327e95fdb75f6add7e911beefae124)

File: src/style/foundation/main.less (hunk 4 above)
Comment (przemvs) [reply src/style/foundation/main.less]: [795a769](https://github.com/wireapp/wire-webapp/pull/15745/commits/795a76903f327e95fdb75f6add7e911beefae124)

File: src/style/modal/certificate-details-modal.less (hunk 2 above)
Comment (przemvs) [reply src/style/modal/certificate-details-modal.less]: [795a769](https://github.com/wireapp/wire-webapp/pull/15745/commits/795a76903f327e95fdb75f6add7e911beefae124)

File: src/script/components/Modals/CertificateDetailsModal/CertificateDetailsModal.test.tsx (hunk 1 above)
Comment (przemvs) [reply src/script/components/Modals/CertificateDetailsModal/CertificateDetailsModal.test.tsx]: [795a769](https://github.com/wireapp/wire-webapp/pull/15745/commits/795a76903f327e95fdb75f6add7e911beefae124)

**Review by atomrc — APPROVED** [review]

**Pull request merged**